Assets/Scripts/Rendering/UsdPathSerializer.cs

```diff
@@ -90,9 +91,9 @@ class UsdCameraXformSample : XformSample {
     var sketchRoot = ExportUsd.CreateSketchRoot();
     m_UsdCamera = new UsdCameraXformSample();
     try {
-      m_Scene = USD.NET.Scene.Create();
+      m_Scene = USD.NET.Scene.Create(path);
```

Comment:
This throws an Argument Null Exception when attempting to record a video.

Reply:
Ah yes... I forgot it could be null for an unsaved file.